include/cquadpack.h

```diff
@@ -9,7 +9,7 @@
 #define epmach     DBL_EPSILON
 #define LIMIT     500
 #define MAXP1     21
-#define Pi      M_PI
+#define Pi      3.141592653589793
```

Comment:
Thanks for the fixes. Could you please use a few more digits, just in case the compiler pre-optimizes some multiplications using 80-bits floating point precision? Thanks!

(`3.14159265358979323846` would be fine)